client/src/components/Ticketing/ticketingmanager/AdminPurchase/AdminCheckout.tsx

```diff
@@ -22,62 +24,64 @@ const stripePromise = loadStripe(pk);
  *
  * @returns {ReactElement}
  */
-export default function CheckoutPage(): ReactElement {
+export default function AdminCheckout(): ReactElement {
+  const location = useLocation();
+  const eventDataFromPurchase = location.state?.eventData || [];
+  console.log('eventDataFromPurchase:', eventDataFromPurchase);
   const navigate = useNavigate();
-  const cartItems = useAppSelector(selectCartContents);
+  const cartItems = location.state?.cartItems || [];
   const discount = useAppSelector(selectDiscount);
-  const donation = useAppSelector(selectDonation);
   const handleBackButton = () => {
     navigate('/ticketing/purchaseticket');
   };
+  const dispatch = useAppDispatch();
```

Comment:
Could you organize this section of code into three parts: hooks, variables with state, and then put the handleBackButton function last. This should make the code a bit easier to decipher. Also we can also remove the `console.log`

Reply:
organized variables and function.
![image](https://github.com/WonderTix/WonderTix/assets/97830813/62737ca6-ce48-4671-aaca-671a905be602)